easybuild/easyconfigs/c/Clang/Clang-18.1.8-GCCcore-13.3.0.eb

```
@@ -1,15 +1,6 @@
-##
-# This file is an EasyBuild reciPY as per https://github.com/easybuilders/easybuild
-#
-# Copyright:: Copyright 2013-2015 Dmitri Gribenko, Ward Poelmans
-# Authors:: Dmitri Gribenko <gribozavr@gmail.com>
-# Authors:: Ward Poelmans <wpoely86@gmail.com>
-# License:: GPLv2 or later, MIT, three-clause BSD.
-# $Id$
-##
-
 name = 'Clang'
 version = '18.1.8'
+easyblock = 'EB_LLVM'
```

`easyblock` usually goes above `name` with an empty line in between:
```suggestion
easyblock = 'EB_LLVM'

name = 'Clang'
version = '18.1.8'
```

---

Done. Also fixing the the CI failure by adding CUDA CCs (copied from another CUDA 12.6 EC)